.github/scripts/pr_body_check.py

```
@@ -0,0 +1,68 @@
+        return
+
+    incomplete_sections = []
+    for section, template_text in template_sections.items():
+        pr_text = pr_sections.get(section, '')
+        # Special handling for Unit Testing section with checkbox
+        if "unit testing" in section.lower():
+            if not has_checked_box(pr_text) and not is_na(pr_text):
+                incomplete_sections.append(f"{section} (checkbox not checked or not marked as N/A)")
+            continue
+        if not normalize(pr_text) or normalize(pr_text) == normalize(template_text):
+            if not is_na(pr_text):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)